data/la/people/Alan-Seabaugh-5032448a-7275-4a02-8dc1-d251cdde8ad9.yml

```diff
@@ -11,6 +11,10 @@ contact_details:
   email: seabaugha@legis.la.gov
   note: District Office
   voice: 318-676-7990
+- address: 9727 Norris Ferry Road;Shreveport, LA 71106-7721
```

Comment:
It looks like this is the new district office, and the other one does not exist anymore.